app/views/companies/edit.html.erb

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
empty line

Reply:
fixed in https://github.com/kanatsanan6/todos-app-on-rails/pull/15/